htlcswitch/failure_test.go

```diff
@@ -0,0 +1,82 @@
+package htlcswitch
+
+import (
+	"encoding/hex"
+	"testing"
+
+	"github.com/btcsuite/btcd/btcec/v2"
+	sphinx "github.com/lightningnetwork/lightning-onion"
+	"github.com/lightningnetwork/lnd/lnwire"
+	"github.com/lightningnetwork/lnd/tlv"
+	"github.com/stretchr/testify/require"
+)
+
+// TestLongFailureMessage tests that longer failure messages can be interpreted
+// correctly.
+func TestLongFailureMessage(t *testing.T) {
+	t.Parallel()
+
+	// Use long 1024-byte test vector from BOLT 04.
+	sessionKeyBytes, _ := hex.DecodeString("4141414141414141414141414141414141414141414141414141414141414141") //nolint:lll
```

Comment:
This could at least be wrapped so the long string is on a new line.

Reply:
Done